wondrous-app/components/Common/Task/modal.tsx

```diff
@@ -773,7 +773,6 @@ export const TaskViewModal = (props: ITaskListModalProps) => {
 
   const [getTaskById] = useLazyQuery(GET_TASK_BY_ID, {
     fetchPolicy: 'network-only',
-    nextFetchPolicy: 'cache-and-network',
```

Comment:
Why do we remove this @juniusfree ?

Reply:
@andywong418 I reverted this change. I thought it's contributing to the issue #2 that I mentioned above.